src/Tools/dotnet-monitor/Auth/ApiKey/MonitorApiKeyConfiguration.cs

```diff
@@ -16,9 +16,9 @@ namespace Microsoft.Diagnostics.Tools.Monitor.Auth.ApiKey
     internal class MonitorApiKeyConfiguration : AuthenticationSchemeOptions
     {
         public bool Configured { get; set; }
-        public string Subject { get; set; }
-        public SecurityKey PublicKey { get; set; }
-        public IEnumerable<ValidationResult> ValidationErrors { get; set; }
-        public string Issuer { get; set; }
+        public string? Subject { get; set; }
+        public SecurityKey? PublicKey { get; set; }
+        public IEnumerable<ValidationResult>? ValidationErrors { get; set; }
```

Comment:
nit: It's usually easier to work with empty collections than null

Reply:
It is, but I'm aiming to minimize code changes in this PR due to the size. That kind of cleanup should be done as separate PRs. See this PR description (https://github.com/dotnet/dotnet-monitor/pull/6924) for the approach I'm taking.